src/cli/cli.cpp

```diff
@@ -580,70 +578,62 @@ template <> otError Interpreter::Process<Cmd("ba")>(Arg aArgs[])
                 SuccessOrExit(error = aArgs[4].ParseAsUint16(port));
             }
 
-            error = otBorderAgentSetEphemeralKey(GetInstancePtr(), aArgs[2].GetCString(), timeout, port);
+            error = otBorderAgentEphemeralKeyStart(GetInstancePtr(), aArgs[2].GetCString(), timeout, port);
         }
         /**
-         * @cli ba ephemeralkey clear
+         * @cli ba ephemeralkey stop
          * @code
-         * ba ephemeralkey clear
+         * ba ephemeralkey stop
          * Done
          * @endcode
          * @par api_copy
-         * #otBorderAgentClearEphemeralKey
+         * #otBorderAgentEphemeralKeyStop
          */
-        else if (aArgs[1] == "clear")
+        else if (aArgs[1] == "stop")
+        {
+            otBorderAgentEphemeralKeyStop(GetInstancePtr());
+        }
+        /**
+         * @cli ba ephemeralkey port
+         * @code
+         * ba ephemeralkey port
+         * 49153
+         * Done
+         * @endcode
+         * @par api_copy
+         * #otBorderAgentEphemeralKeyGetUdpPort
+         */
+        else if (aArgs[1] == "port")
         {
-            otBorderAgentClearEphemeralKey(GetInstancePtr());
+            OutputLine("%u", otBorderAgentEphemeralKeyGetUdpPort(GetInstancePtr()));
         }
         /**
          * @cli ba ephemeralkey callback (enable, disable)
          * @code
          * ba ephemeralkey callback enable
          * Done
-         * ba ephemeralkey set W10X1 5000 49155
+         * ba ephemeralkey start W10X10 5000 49155
```

Comment:
```suggestion
         * @endcode
         * @code
         * ba ephemeralkey start W10X10 5000 49155
```

Each command needs to be in its own block